docs/reference/modules/discovery/zen.asciidoc

```diff
@@ -82,6 +82,12 @@ serves as a protection against (partial) network failures where node may unjustl
 think that the master has failed. In this case the node will simply hear from
 other nodes about the currently active master.
 
+If the setting `discovery.zen.master_election.filter_client` is `true`, pings from client nodes (or non-data,
+non-master-eligible nodes) will be ignored during master election; the default value is `true`. If the setting
```

Comment:
Thanks for the improvements @debadair! Since "acknowledged" is a variant of "acknowledgement" which has a [technical meaning in networking](https://en.wikipedia.org/wiki/Acknowledgement_(data_networks)) that is not in play here I think that we shouldn't use that term.